pkg/action/action.go

```diff
@@ -379,9 +379,14 @@ func (cfg *Configuration) Init(getter genericclioptions.RESTClientGetter, namesp
 		clientFn:  kc.Factory.KubernetesClientSet,
 	}
 
+	if helmDriver == "" {
+		helmDriver = "secrets"
+	}
+
+	log("init: using %s storage driver", helmDriver)
```

Comment:
Don't we want instead to say that is was not set so we default at `secrets` ?

Reply:
good idea, pushed changes